doc/move_group_interface/launch/move_group.launch.py

```diff
@@ -159,7 +159,11 @@ def generate_launch_description():
 
     # Load controllers
     load_controllers = []
-    for controller in ["panda_arm_controller", "joint_state_controller"]:
+    for controller in [
+        "panda_arm_controller",
+        "panda_hand_controller",
+        "joint_state_broadcaster",
```

Comment:
If we want to change `joint_state_controller` to 'joint_state_boradcaster` we also need to change it from `moveit_resources/panda_moveit_config/config/panda_ros_controllers.yaml`. This will affect all the tutorials / demos / applications that is currently using it. I say we wait for it and change in either rolling or galactic.

Reply:
I reverted this change.